examples/trigger/variables.tf

```diff
@@ -37,8 +25,31 @@ variable "resource_group_name" {
   type        = string
 }
 
-variable "provision_namespace" {
-  description = "Provision Namespace"
+variable "is_new_trigger" {
+  description = "Provision Triger"
   type        = bool
-  default     = false
+  default     = true
 }
+
+variable "trigger_name" {
+  description = "Name of Trigger."
+  type        = string
+}
+
+variable "trigger_feed" {
```

Comment:
expand the contents of trigger_feed like name, parameters etc

Reply:
Fixed